docs/ci-operations-group.md

```diff
@@ -0,0 +1,140 @@
+# KubeVirt CI Operations Group
+
+
+## Goal of this document
+
+The current group consists only of a couple of people who mostly are not only handling CI but need to take other tasks as well. This document is intended to make clear what the primary responsibilities are in order to avoid overloading the members of this group with tasks.
+
+
+## Primary members
+
+<table>
+  <tr>
+   <td></td>
+   <td></td>
+   <td></td>
+   <td colspan="5" ><b>Responsibilities / Area of interest</b></td>
+  </tr>
+  <tr>
+   <td><b>Human</b></td>
+   <td><b>Availability</b></td>
+   <td><b>TZ</b></td>
+   <td><b>BM</b></td>
+   <td><b>Operations</b></td>
+   <td><b>Prow</b></td>
+   <td><b>kubevirtci</b></td>
+   <td><b>SIG Support</b></td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/brianmcarey">Brian Carey</a></td>
+   <td>Mon-Fri</td>
+   <td>GMT+1</td>
+   <td>Pri</td>
+   <td>Pri</td>
+   <td>Pri</td>
+   <td>Backup</td>
+   <td>Backup</td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/dhiller">Daniel Hiller</a></td>
+   <td>Mon-Fri</td>
+   <td>GMT+2</td>
+   <td>Backup</td>
+   <td>Sec</td>
+   <td>Pri</td>
+   <td>Backup</td>
+   <td>Pri</td>
+  </tr>
+  <tr>
+   <td><a href="https://github.com/enp0s3">Igor Bezukh</a></td>
+   <td>Sun-Thu</td>
+   <td>GMT+3</td>
+   <td></td>
+   <td>Backup</td>
```

Comment:
We might want to explain what `Pri, Sec, Backup` means just to remove any doubts.

Reply:
I've replaced those with the full words, I think that would make it clear already. WDYT?